### PR TITLE
Newbranch file.txt is done

### PR DESCRIPTION
**Pull request by antonsme**

### CODE REVIEW

File: LABS/file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello gleb
```
Comment (Skilsar) [inline LABS/file.txt]: Update name to uppercase

**Review by Skilsar — CHANGES_REQUESTED** [review]
@antonsme  Fix some error

**Comment by antonsme** [comment]
@Skilsar изменил короче

**Review by Skilsar — APPROVED** [review]
All good

**Pull request merged**